packages/ui/react-ui-kanban/src/components/Kanban.tsx

```diff
@@ -84,6 +95,7 @@ export const Kanban = ({ model, onAddCard, onRemoveCard }: KanbanProps) => {
                             label={t('card drag handle label')}
                           />
                         </StackItem.DragHandle>
+                        <AttentionGlyph attended={selectedItems.has(card.id)} />
```

Comment:
I know this is a pattern we have elsewhere but I don't think it's helpful as implemented here for a couple reasons:
1. the glyph on its own isn't noticeable enough
2. there's no glyph in the sidebar to pair with it

I could see the glyph being useful at some point here but I'd say in addition to a selection ring for the same reason that we added the selection background colour change in the navtree, it's just much clearer what's going on.

Reply:
I think we must leave rings as exclusive signifiers of focus, otherwise this will get very confusing.

I think we could also change the background color as an added signifier, especially to remain consistent with the navtree, but it should still come with the relatedness signifier, otherwise we should reconsider using the relatedness signifier at all.

Reply:
> there's no glyph in the sidebar to pair with it

The panel of the R1 should display such a glyph anyway in order to carry out a design consistent with the rest of the app, I would actually recommend following through on that.

Reply:
If we don’t have consensus that this should be consistent with the navtree, then it should be consistent with table by using checkboxes.

There is not sufficient cause to implement a third way of signifying `current` state.

Reply:
My feedback is that the glyph signifier as demonstrated above is not clear enough as an indication of selection, I don't have too strong an opinion on how that's addressed, ring is just a proposed option.

Reply:
Agreed that it is too subtle a signifier on its own. I think there are two reasonable options:
- use checkboxes like table
- use a background change and an associated glyph like navtree

Personally I would prefer this use the same signifiers as navtree since I think selection state in table has conflated selection with `current`, which are orthogonal and should be signified separately, i.e. table should also adopt the same pattern as navtree eventually.

Reply:
I get the feeling this entire side-bar editing pattern is a stop-gap measure. Given that we're pushing to get *something* working I think we should go with the current implementation and circle back with more design thought and a more considered ontology later.

The way this is written, it's very orthogonal and doesn't complect either the table or kanban implementations. Very easy to rip back out.